.gitattributes

```diff
@@ -3,8 +3,8 @@
 # - tell git to guess whether files are text or binary
 *                       text=auto
 
-# Mark .pnp.js as binary to prevent git from trying to merge it
-/.pnp.js                binary linguist-generated
+# Mark .pnp.cjs as binary to prevent git from trying to merge it
+/.pnp.cjs                binary linguist-generated
```

Comment:
I think we need to keep both here as git is showing a massive diff on this PR

Reply:
🤔 Looks like Git is still showing the massive diff.

Reply:
Looking at the output of `git pull` this doesn't count as a rename so it makes sense, I suppose git can't tell because its told the file is a binary while the GitHub UI slightly ignores that fact

```
 .pnp.cjs                                           | Bin 0 -> 2384524 bytes
 .pnp.js                                            | Bin 2434101 -> 0 bytes
 create mode 100755 .pnp.cjs
 delete mode 100755 .pnp.js
```